test/e2e/windows/gmsa_full.go

```diff
@@ -163,6 +169,73 @@ var _ = SIGDescribe("[Feature:Windows] GMSA Full [Serial] [Slow]", func() {
 				return true
 			}, 1*time.Minute, 1*time.Second).Should(gomega.BeTrue())
 		})
+
+		ginkgo.It("can read and write file to remote folder", func() {
+			defer ginkgo.GinkgoRecover()
+
+			ginkgo.By("finding the worker node that fulfills this test's assumptions")
+			nodes := findPreconfiguredGmsaNodes(f.ClientSet)
```

Comment:
All of this logic is the same as the `GMSA support` logic. Can we hoist this to a `before` clause that will set up the cluster with the web hook once (and do the clean up in a `after`)?

Reply:
can you create an issue to track this clean up.  I think it is important clean up to be done

Reply:
Sure, I created it and assigned to myself. https://github.com/kubernetes/kubernetes/issues/108649